native_client/ctcdecode/scorer.cpp

```diff
@@ -56,13 +54,26 @@ Scorer::Scorer(double alpha,
   setup(lm_path, trie_path);
 }
 
+Scorer::Scorer(double alpha,
+               double beta,
+               const std::string& lm_path,
+               const std::string& trie_path,
```

Comment:
We pass `lm_path` and `trie_path` on the constructor as well as on `init` (above) ?

Reply:
Would it make sense to make them members of `Scorer` like `alphabet_` ?

Reply:
I don't understand the suggestion. What would making them members accomplish?

Reply:
Is there any reason we have two entry points to set those vlaues ?

Reply:
We don't. `init` is private.

Reply:
Right, then I think it would be simpler to just have `lm_path_ = lm_path` and have `init()` using them

Reply:
They aren't used anywhere outside of setup_lm. There's no reason to inflate the object size.

Reply:
r=me then

Reply:
I've merged init and setup to avoid this problem.